webapp/src/modules/nft/sagas.ts

```diff
@@ -94,7 +94,8 @@ function* handleFetchNFTRequest(action: FetchNFTRequestAction) {
     ]: AwaitFn<typeof vendor.nftService.fetchOne> = yield call(
       [vendor.nftService, 'fetchOne'],
       contractAddress,
-      tokenId
+      tokenId,
+      { status: options?.status }
```

Comment:
```suggestion
    options
```
can we use the whole options as parameter here? I believe it looks a bit cleaner and we avoid the `{ status: undefined }` case. WDYT?

Reply:
It's better, you're right, changing it.